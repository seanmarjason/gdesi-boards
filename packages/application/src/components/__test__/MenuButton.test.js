import { render } from '@testing-library/react'
import MenuButton from '../MenuButton';

it('MenuButton renders correctly', () => {
  const { container } = render(<MenuButton />)
  expect(container).toMatchSnapshot()
})
