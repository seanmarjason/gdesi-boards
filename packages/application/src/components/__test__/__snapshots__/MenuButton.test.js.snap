// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuButton renders correctly 1`] = `
<div>
  <span
    class="MuiBadge-root css-j35v34-MuiBadge-root"
  >
    <button
      class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-155wbr6-MuiButtonBase-root-MuiIconButton-root"
      id=":r0:"
      tabindex="0"
      type="button"
    />
    <span
      class="MuiBadge-badge MuiBadge-dot MuiBadge-invisible MuiBadge-anchorOriginTopRight MuiBadge-anchorOriginTopRightRectangular MuiBadge-overlapRectangular MuiBadge-colorError css-dn5ucj-MuiBadge-badge"
    />
  </span>
</div>
`;
